Import Tezos X doc (#436)
etherlinkcom/docs: docs/overview/index.md

---
title: What is Etherlink? # tx
slug: /
---

# What is Etherlink<!--TX-->?

Etherlink<!--TX--> is an EVM-compatible, non-custodial execution layer enshrined in the Tezos protocol and powered by [Tezos Smart Rollup technology](https://tezos.com/developers/smart-rollups/), which settles back to the Tezos consensus layer (Tezos Layer 1) for security.

It enables seamless integration with existing Ethereum tools, including wallets and indexers, and facilitates asset transfers to and from other EVM-compatible chains.

> Since kernel upgrade 7, Etherlink<!--TX--> also exposes a **Michelson interface** (previously called Tezlink), thus implementing a **single blockchain** that is **addressable through two interfaces**: the **EVM interface** and the new **Michelson interface** (previously called Tezlink).

Reviewer: "(previously called Tezlink)" appears twice in the same sentence — one mention is enough. Also, this paragraph is the only place in the whole site that mentions **Tezos X**, without saying what it is or linking anywhere. For a reader who arrives without context, it's unclear whether Tezos X is a different product, a rebrand, or a roadmap name. One sentence (or a link) clarifying the Etherlink ↔ Tezos X relationship would help a lot.

Author: done

Reviewer: Verified ✓ — the duplicate "(previously called Tezlink)" is gone and Tezos X now only survives as an HTML comment, consistent with the branding decision. Resolvable.

---

> title: What is Etherlink? # tx

Reviewer 2: I think this is changing the index of the doc too much tbh and i'm not sure to understand why.

You should have changed the first paragraph to generalize it and be done with it IMO.

Author: I can minimize the changes WRT the old version, just mentioning the dual interface and the NAC here, and leaving the details for the other pages in this Overview section. It depends if we want to emphasize continuity (via minimal changes) or the new features. I'm open to both, maybe anybody else has an opinion?

Author: Check the last commit "minimize changes to the landing page"

slug: /
---

# What is Etherlink<!--TX-->?

> Etherlink<!--TX--> is a fast, non-custodial execution layer enshrined in the Tezos protocol, which settles back to the Tezos consensus layer (Tezos Layer 1) for security.

Reviewer 2: ```suggestion
Etherlink<!--TX--> is a fast, non-custodial execution layer enshrined for the Tezos ecosystem and powered by [Tezos Smart Rollup technology](https://tezos.com/developers/smart-rollups/).
```

Reviewer 2: I am not sure why you dropped the “Fast, fair and (nearly) free” motto that was present in the previous version.

Author: The motto is kept in all detail lower in the page, but right, let's introduce it right ahead:
```suggestion
Etherlink<!--TX--> is a non-custodial execution layer enshrined in the Tezos protocol and powered by [Tezos Smart Rollup technology](https://tezos.com/developers/smart-rollups/), which settles back to the Tezos consensus layer (Tezos Layer 1) for security.

It brings a *fast, fair, and (nearly) free* user and developer experience.
```

> Since kernel upgrade 7, Etherlink<!--TX--> exposes a **single blockchain** that is **addressable through two interfaces**: an **EVM interface** (available since early 2024 under the name "Etherlink") and a new **Michelson interface** (previously called Tezlink).

Reviewer 2: ```suggestion
Etherlink<!--TX--> exposes a **single blockchain** that is **addressable through two interfaces**: an **EVM interface** and a **Michelson interface**.
```

Author: Sure, we can just say what Etherlink is now, but in order to explain to current users why there are so many changes, isn't it useful to say that the whole changes are brought in this upgrade? So people can distinguish the Etherlink-before and the Etherlink-now.

Author: Maybe makes more sense in the new minimal-changes version, take a look.

> In that, kernel upgrade 7 constituted an important milestone in the Tezos X roadmap, securing the way to the near-term Tezos X launch.

Reviewer 2: ```suggestion
```

Author: François (I cannot find his tag here) suggested to mention this as a milestone, check with him, I'm open to any suggestions.

Reviewer: I would remove it at the moment. While this is true, the Tezos X roadmap, extended to take into account end-users has not been made public yet.

Author: commented out (the link I included was indeed the old roadmap not including end-users)

> To enable the seamless integration of the two ecosystems in this single blockchain, Etherlink<!--TX--> provides **Native Atomic Composability** (sometimes shortened as NAC): smart contracts in one interface can call contracts in the other within a single atomic transaction. From an economical perspective, the interface used to interact with the chain doesn't matter, it becomes only a technical detail.

Reviewer 2: Why “economical perspective”?

Author: François's phrasing, open to change, but check with him. I understand it as: the two worlds share the same native token. (this is explicit in the next sentence)

Reviewer: Maybe we could remove this phrase and just mention the two interfaces share the same native token?

Author: done (note that the phrase is now in page overview/architecture.md)
